substrate/frame/paged-list/src/paged_list.rs

```diff
@@ -432,6 +428,7 @@ pub(crate) mod mock {
 #[cfg(test)]
 mod tests {
 	use super::mock::*;
+	use frame::testing_prelude::{TestExternalities, *};
```

Comment:
```suggestion
	use frame::testing_prelude::*;
```

no need to import `TestExternalities`